HOW_TO

```diff
@@ -0,0 +1,12 @@
+How to test the OpenFoam internal interfaces implementation
```

Comment:
Could the contents of the HOW_TO be included in the README or other documentation?  It would be best to include in the documentation how the user uses the INTERNAL and FILE-IO wrappers and how to build either the INTERNAL or FILE-IO wrappers.